core/validator.go

```diff
@@ -59,37 +60,24 @@ func (v *chunkValidator) validateBlobQuorum(quorumHeader *BlobQuorumInfo, blob *
 		return nil, nil, nil, errors.New("number of chunks does not match assignment")
 	}
 
-	chunkLength, err := v.assignment.GetChunkLengthFromHeader(operatorState, quorumHeader)
-	if err != nil {
-		return nil, nil, nil, err
-	}
-
 	// Validate the chunkLength against the quorum and adversary threshold parameters
-	numOperators := uint(len(operatorState.Operators[quorumHeader.QuorumID]))
-	minChunkLength, err := v.assignment.GetMinimumChunkLength(numOperators, blob.BlobHeader.BlobCommitments.Length, quorumHeader.QuantizationFactor, quorumHeader.QuorumThreshold, quorumHeader.AdversaryThreshold)
-	if err != nil {
-		return nil, nil, nil, err
-	}
-	params, err := GetEncodingParams(minChunkLength, info.TotalChunks)
-	if err != nil {
-		return nil, nil, nil, err
-	}
-
-	if params.ChunkLength != chunkLength {
```

Comment:
do we need to keep this check?

Reply:
Hmm... yeah, I think we need this is in order to ensure the chunkLength is a power of 2.